### PR TITLE
Improve scanner UI

### LINKED ISSUE #9627: Remove vul database update timestamp from UI
As the Clair scanner is deployed with out of tree way and Clair does not provide API to get the vul DB updated timestamp info, the related label should be removed from the UI @`Vulnerability` section.

### PR DESCRIPTION
**Pull request by AllForNothing**
1.Modify vulnerabilities report bar css
image
2.Fix permission bugs
3.Fix "withClair" flag bugs
4.Fix unstable UT case
5.Fix #9627
Signed-off-by: sshijun <sshijun@vmware.com>

### CODE REVIEW

File: src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss [hunk 1]
```diff
@@ -225,51 +217,74 @@ hr {
...
-        border-radius: 50%;
-        height: 20px;
-        width: 20px;
-        line-height: 20px;
-        font-size: 14px;
-        text-align: center;
-    }
+    justify-items: center;
+    border-radius: 50%;
+    height: 30px;
```
Comment (zhoumeina) [inline src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss]: define a variable for it.

File: src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss [hunk 2]
```diff
@@ -225,51 +217,74 @@ hr {
...
+    width: 30px;
+    line-height: 30px;
 }
 
-.level-border {
-    border:1px solid #f8b5b4;
+.inner {
+    margin: auto;
+    height: 22px;
```
Comment (zhoumeina) [inline src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss]: 22px should be defined as a variable.

File: src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss [hunk 3]
```diff
@@ -225,51 +217,74 @@ hr {
...
-    }
+    justify-items: center;
+    border-radius: 50%;
+    height: 30px;
+    width: 30px;
+    line-height: 30px;
 }
 
-.level-border {
-    border:1px solid #f8b5b4;
```
Comment (zhoumeina) [inline src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss]: should format.

File: src/portal/lib/src/vulnerability-scanning/scanning.scss [hunk 4]
```diff
@@ -144,31 +138,34 @@ hr{
...
     background:#e64524!important;
     color:#621501!important;
-    border:1px solid #f8b5b4!important;
+    border:1px solid #fff!important;
 }
 .label-medium {
     background-color: orange;
     color:#621501;
-    border:1px solid #f8b5b4;
+    border:1px solid #fff;
```
Comment (zhoumeina) [inline src/portal/lib/src/vulnerability-scanning/scanning.scss]: border should be in a single style.

File: src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.scss [hunk 5]
```diff
@@ -7,3 +7,9 @@
 .clr-control-label {
   width: 9rem !important;
 }
+.width-10rem {
```
Comment (zhoumeina) [inline src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.scss]: can be removed.

File: src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.spec.ts [hunk 6]
```diff
@@ -57,32 +61,48 @@ describe('NewScannerFormComponent', () => {
...
+    nameInput.value = "test1";
+    nameInput.dispatchEvent(new Event('input'));
+    nameInput.blur();
+    nameInput.dispatchEvent(new Event('blur'));
+    let el = null;
+    setTimeout(() => {
+      el = fixture.nativeElement.querySelector('#name-error');
+      expect(el).toBeTruthy();
+    }, 20000);
+    tick(20000);
```
Comment (zhoumeina) [inline src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.spec.ts]: Suggest use async done to expect the value

File: src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss (hunk 1 above)
Comment (AllForNothing) [reply src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss]: done

File: src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss (hunk 3 above)
Comment (AllForNothing) [reply src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss]: fixed

File: src/portal/lib/src/vulnerability-scanning/scanning.scss (hunk 4 above)
Comment (AllForNothing) [reply src/portal/lib/src/vulnerability-scanning/scanning.scss]: fixed

File: src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.scss (hunk 5 above)
Comment (AllForNothing) [reply src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.scss]: fixed

File: src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.spec.ts (hunk 6 above)
Comment (AllForNothing) [reply src/portal/src/app/config/scanner/new-scanner-form/new-scanner-form.component.spec.ts]: "async done" is not suitable for this case.

File: src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss (hunk 2 above)
Comment (AllForNothing) [reply src/portal/lib/src/vulnerability-scanning/result-tip-histogram/result-tip-histogram.component.scss]: fixed

**Pull request merged**